createdisk.sh

```diff
@@ -24,10 +24,10 @@ fi
 BASE_OS=${BASE_OS:-rhcos}
 OPENSHIFT_VERSION=$(${JQ} -r .clusterInfo.openshiftVersion $INSTALL_DIR/crc-bundle-info.json)
 
-# CRC_VM_NAME: short VM name to use in crc_libvirt.sh
+# SNC_PRODUCT_NAME: short VM name to use in crc_libvirt.sh
```

Comment:
It's not really a "short VM name" anymore, and `crc_libvirt.sh` has been gone for a very long time, would be worth updating the comment at the same time.